### DISCUSSION TITLE
Icon in Series page to denote a series has been read fully

### DISCUSSION THREAD

**Discussion by jncmney** [Kareadita/Kavita#3447] [Ideas]
### Idea Description

In the Series page, there's no way to quickly see if we've read the entire series. The only way is to look at all the issues to confirm they don't have the green triangle in the upper right corner.

It would be great to add an icon to denote the series has been read in its entirety. 

It could be in the line under the title, next to the "Time to Read" icon, like this:

<img width="1020" alt="image" src="https://github.com/user-attachments/assets/d07388b6-c9bb-42bf-8565-c8f696613e44">

Use Cases:

- If we can't remember if we've read a series. It just happened to me (which is what sparked this FR). I saw a series by an artist I like and couldn't remember if I'd read it, since I've read a ton of her stuff. I found the series in Kavita, but it has dozens of issues, so I had to look at them all to see they didn't have the green triangle. 
- If we've read most, but for some reason left the series halfway through. I have Smart Filters to find these, but when I'm browsing my library it'd be nice to have this icon.

### Idea Category

Web UI

### Duration of Using Kavita

About a year

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by rix8675** [comment]
you could also have something this on the home/library navigation pages.
- not started, just have the orange triangle, or have the remaining chapters/volumes number.
- incomplete, have a remaining chapters/volumes number.(basically started but not finished)
- completed, no triangle or number.

once in the series you could do a few things.
1. have the kavita icon(green color in the background) in the middle of the artwork fill in either radial or vertical based on a percentage amount or something along those lines.
2. basically the same as number one but have the green fill in behind the continue button based on how far you've progressed as a percentage ,or use little generic book images instead of just a green bar.
